pwa/src/components/configurations/configurations.tsx

```diff
@@ -44,7 +48,31 @@ export default function ConfigurationsTable() {
       });
   };
 
-  return (
+  const handleExport = () => {
+    fetch(`${context.adminUrl}/export/all`, {
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: "Bearer " + sessionStorage.getItem("jwt"),
+      },
+    }).then((response) => {
+      response.text().then(function (text) {
+        download("export.yaml", text, "text/yaml");
+      });
+    }).catch((error) => {
+      console.log('Error:', error)
+      setAlert(null);
+      setAlert({ type: 'danger', message: error.message });
+    });
+  };
+
+  return (<>
+    {
+      alert !== null &&
+      <FlashMessage duration={5000}>
+        <Alert alertClass={alert.type} body={function () { return (<>{alert.message}</>) }} />
+      </FlashMessage>
```

Comment:
Deze logica én de `<button/>` mag naar de `page` i.p.v. dat het in een component staat.